### ISSUE TITLE
The mod keeps showing blocks into my hotbar.

### ISSUE THREAD

**Issue by ThatExplo** [maruohon/tweakeroo#192]
Whenever I pickup stone or dirt it gets shoved into my last hotbar slot.
I removed all mods other than tweakeroo so I know its this one.
I tried going trough the settings however none of the options did anything.

**Comment by maruohon** [comment]
Do you mean that any picked up items go to the last empty/valid hotbar slot instead of the first?

There should not be anything in my mods that would change stuff like that, and that does not happen to me in my test instance when having my mods installed. It behaves the same as in vanilla when picking up items from the ground to an empty inventory, they go to hotbar slots 1 and 2 first.

**Comment by ThatExplo** [comment]
No, a single dirt/cobblestone block always shoves itself into my last hotbar slot, aslong as I have one in my inventory.
If I try to take it out it gets replaced by a different dirt/stone block, same with dropping it.
( Aslong as I have any stone/dirt in my inventory )
It started happening after I went trough the settings for the first time.

**Comment by SkeletalPrints** [comment]
Go back through your setting and reset them all. But by this time you have probably already solved the problem.

**Comment by maruohon** [comment]
Was this solved? Are you in 1.16.x or 1.12.2? I don't see how any feature in the 1.16 version could even be doing that, unless you have some forked version of the mod with more features.

**Comment by Mistera91** [comment]
Hi, just found this issue, I have the same problem (running the latest tweakeroo build for 1.20.4).
I’m fine with trying to debug this issue (since the reproduction steps are hard to find, I use the mod normally and have downloaded it recently), just tell me what to do in order to show debug infos, or settings to change.
So far, I’ve tried to disable everything in the « Generic » tab, and to reset every setting. The issue was still here in both cases.
Reinstalling the mod / Removing the configuration files would probably work, but I try to avoid doing it until we can find where this bug is coming from (potentially from another of my mods, or multiple of them) (I have 40+ mods).

**Comment by maruohon** [comment]
@Mistera91  Re-installing the mod does nothing at all, at least if you just put the exact same mod file back again. And you can test config resetting by moving away the `tweakeroo.json` file from the `config` directory instead of deleting it (and do it while the game is closed, otherwise it will just get saved back when you leave the world or close the game).

If it's actually something in Tweakeroo (which I still can't think of what it would be), then it would be something in the Tweaks category, or possibly the slot sync workaround from Generic (which shouldn't do anything though unless you use fast right click or fast block placement).

**Comment by Mistera91** [comment]
> moving away the `tweakeroo.json` file from the `config` directory instead of deleting it (and do it while the game is closed, otherwise it will just get saved back when you leave the world or close the game).

I just tried this, same problem, still here even though the config file is not yet created. I changed a random setting to create the file and the problem was still here.

**Comment by maruohon** [comment]
@Mistera91 Can you test this in a separate test instance where you have only malilib and Tweakeroo installed? And which exact mod version are you using? If it happens in that test instance, then can you record a video clip showing how it happens, i.e. how to reproduce the issue on my end?

**Comment by Mistera91** [comment]
I figured out that I had `allowInventory` set to `true` in Baritone, and that caused this issue. Sorry for the inconvenience.

**Comment by Janys159** [comment]
> I figured out that I had `allowInventory` set to `true` in Baritone, and that caused this issue. Sorry for the inconvenience.

bro thanks so much, i have been searching the answer to that so long
